docker-dashboard-front-end/src/dashboard/Container.js

```diff
@@ -1,33 +1,44 @@
 import React, {useState, useEffect} from 'react';
-import {useParams} from "react-router";
+import {useParams, useHistory} from "react-router";
 import { w3cwebsocket as W3CWebSocket } from "websocket";
 import {Button, Grid} from "@material-ui/core";
 import {Terminal} from 'xterm';
 import 'xterm/css/xterm.css'
 import Box from "@material-ui/core/Box";
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import IconButton from '@material-ui/core/IconButton';
 
 function Container() {
     const [connect, setConnect] = useState(false);
-    const { id } = useParams();
+    const history = useHistory()
+    const { jobuuid, id } = useParams();
     const term = new Terminal();
     var protocol = window.location.protocol.replace('http', 'ws');
     var hostname = window.location.hostname === 'localhost' ? 'localhost:3001' : window.location.hostname;
     var client;
     const streamLogs = () => {
         term.open(document.getElementById('terminal'));
-        client = new W3CWebSocket(`${protocol}//${hostname}/container/${id}/logs`);
+        client = new W3CWebSocket(`${protocol}//${hostname}/${jobuuid}/container/${id}/logs`);
         client.onopen = () => client.send('logs');
-        client.onmessage = (data) => {
-            term.writeln(data.data)
+        client.onmessage = function (event) {
+            term.writeln(event.data)
+        }
+        client.onerror = (error) => {
+            console.error(error)
         }
     }
     //eslint-disable-next-line
-    useEffect(() => connect ? streamLogs() : null, [connect])
+    useEffect(() => connect ? streamLogs() : null, [connect]);
     return(
         <div>
+            <IconButton onClick={() => history.goBack()}>
```

Comment:
Can this be an href rather than history so we explicitly know the page it is returning to